lib/iris/fileformats/pp.py

```diff
@@ -1955,23 +1955,33 @@ def _convert_constraints(constraints):
     constraints = iris._constraints.list_of_constraints(constraints)
     pp_constraints = {}
     unhandled_constraints = False
+
+    def make_func(stashobj):
```

Comment:
i'm happy for it to stay as a function  if there's reason for it for functional reasons. It is clear what it does and it's pretty close to the call in the code base.

i'd prefer to see the function named with a leading underscore, to indicate its intended privacy